chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -662,6 +662,13 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
             ::-webkit-scrollbar-thumb:hover {
                 background: ${scrollbarProps.thumbHoverColor};
             }
+
```

Comment:
this already exist here -> https://github.com/microsoft/omnichannel-chat-widget/blob/635e2559f838936a3729eaf3718818669fca1d7d/chat-widget/src/components/webchatcontainerstateful/WebChatContainerStateful.tsx#L177

Reply:
Fixed css in [WebChatContainerStateful.tsx] and removed css from LiveChatWidgetStateful.tsx